### PR TITLE
[Dominion] Add AUL logo and land stories to Kenya instance

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description

- [x] Add AUL logo on CountryPartner component
- [x] Add Kenya land stories to showcase

Fixes # (issue)

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Screenshots
Screenshot from 2019-06-26 12-31-51
Screenshot from 2019-06-27 11-03-13

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: dominion/dominion_ui/build/asset-manifest.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
-  "main.js": "/static/js/main.92678822.chunk.js",
-  "main.js.map": "/static/js/main.92678822.chunk.js.map",
-  "static/js/1.2034513b.chunk.js": "/static/js/1.2034513b.chunk.js",
-  "static/js/1.2034513b.chunk.js.map": "/static/js/1.2034513b.chunk.js.map",
+  "main.js": "/static/js/main.29a670af.chunk.js",
```
Comment (kilemensi) [inline dominion/dominion_ui/build/asset-manifest.json]: I'm serious not liking having `build` folder checked in git :cry:

File: dominion/dominion_ui/src/components/CountryPartners/index.js [hunk 2]
```diff
@@ -46,10 +47,14 @@ const styles = theme => ({
     [theme.breakpoints.up('md')]: {
       padding: '2rem 1rem'
     }
+  },
+  imageGrayScale: {
+    WebkitFilter: 'grayscale(100%)' /* Safari 6.0 - 9.0 */,
+    filter: 'grayscale(100%)'
```
Comment (kilemensi) [inline dominion/dominion_ui/src/components/CountryPartners/index.js]: Isn't it better to `grayscale` the logo in Photoshop or GIMP i.e offline? These filters are cool if the image we have to show in `grayscale` in uploaded at runtime or something.

File: dominion/settings.py [hunk 3]
```diff
@@ -218,25 +218,39 @@
...
+        'index': 0,
+        'title':  'Deforestation: 48 Years of Kenya\'s Unspoken Disaster',
+        'author': 'Africa Uncensored',
+        'brief':  'Deforestation in Kenya, how did it happen? Who is responsible?',
+        'link':   '',
+        'date': '05 Apr 2019',
+        'mediaSrc':   'https://www.youtube.com/embed/PDhuK4KrIoU',
+        'media': 'iframe',
+        'country': 'kenya'
+    },
```
Comment (kilemensi) [inline dominion/settings.py:232]: These should move to a gSheet... On a different story/PR so we don't hold this PR back of course.

File: dominion/settings.py [hunk 4]
```diff
@@ -273,36 +293,55 @@
...
         'author': 'Poloko Tau',
         'brief':  'In Matiwaneskop near Ladysmith, KwaZulu-Natal, the community is refusing to be ruled by Inkosi Ntandoyenkosi Shabalala. They say that, when their forefathers bought the land in the 1800s, they had no chief and were just a group of 120 ordinary people.',
         'link':   'https://city-press.news24.com/News/community-vs-chief-the-story-of-a-struggle-for-land-20180624',
-        'image':    STATIC_URL + 'img/showcase/showcase7.jpg',
-        'date': '24 Jun 2018'
-    }
+        'mediaSrc':    STATIC_URL + 'img/showcase/showcase7.jpg',
+        'date': '24 Jun 2018',
+        'media': 'img',
+        'country': 'south-africa'
```
Comment (kilemensi) [inline dominion/settings.py]: Isn't using country codes better than country slugs? e.g. country codes are standardized (ISO) while with slugs, you have to check how each country name was _slugified_.

**Review by kilemensi — APPROVED** [review]
:+1: @KhadijaMahanga 

I'm not sure about the logo though... looks too dark and those square borders don't help. Isn't there one that's just a bit more plain?

File: dominion/settings.py (hunk 3 above)
Comment (KhadijaMahanga) [reply dominion/settings.py:232]: @kilemensi alright. I will create a story on PT

**Comment by kilemensi** [comment]
@KhadijaMahanga since we're already planning on moving UI to its own repo, I think lets just leave the `build` folder in git for now.

So revert the `gitignore` change, update the screenshot with the new logo and merge. :+1:

**Pull request merged**